samples/boards/96b_argonkey/microphone/src/main.c

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2018 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <zephyr.h>
+#include <misc/printk.h>
+
+#include <gpio.h>
+#include <led.h>
+#include <i2s.h>
+#include <i2c.h>
+#include <spi.h>
```

Comment:
Do you need to include i2s, i2c, spi? I don't seem them being used directly in the sample.

Reply:
Correct.
i2c.h was useless even before, while i2s.h and spi.h became useless with this sample
new version which uses dmic i/f.

Reply:
So, I removed them.